ManagementPacks/Community.DataOnDemand/Scripts/Watch-TcpConnections.ps1

```diff
@@ -0,0 +1,539 @@
+﻿<#
+.SYNOPSIS
+    Get information about TCP connections over the specified number
+    of minutes.
+.DESCRIPTION
+    This script repeatably obtains information about network connections
+	at the host and filters and formats the output in a CSV form friendly
+	to automated consumption.
+.PARAMETER Interval
+    The number of seconds to monitor for
+.PARAMETER Format
+    Permitted values: text, csv, json
+.EXAMPLE
+	PS > .\Watch-TcpConnections.ps1 -format csv
+	Returns netstat information in the default CSV format.
+.NOTES
+    Copyright 2017 Squared Up Limited, All Rights Reserved.
+.LINK
+	https://www.squaredup.com
+#>
+[Diagnostics.CodeAnalysis.SuppressMessageAttribute("PSAvoidUsingWriteHost", "")]
+Param(
+
+    [int]$Interval = 60,
+
+    [ValidateSet("text","csv","csvEx","json","list")]
+    [string] $Format = "csv"
+)
+
+#Requires -Version 2.0
+Set-StrictMode -Version 2.0
+
+$source = @'
```

Comment:
I did discuss this with @shadeon prior to creating the PR... I admit it's a slight inconvenience having to copy/paste the contents of the $source variable between PoSH and VS.NET, but I have haredly ever had to do this (i.e. I developed all the functionality in C# in VS.NET and copied it once)